lib/cc/engine/analyzers/engine_config.rb

```diff
@@ -27,6 +27,12 @@ def concurrency
           config.fetch("config", {}).fetch("concurrency", 2).to_i
         end
 
+        def filters_for(language)
+          fetch_language(language).fetch("filters", []).map { |s|
+            Sexp::Matcher.parse s
+          }
```

Comment:
What does the `s` variable represent? Does it stand for string? What do you think of fleshing it out to "filter" if we're mapping over filters?